config/environments/production.rb

```diff
@@ -14,16 +14,9 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
-  # Enable Rack::Cache to put a simple HTTP cache in front of your application
-  # Add `rack-cache` to your Gemfile before enabling this.
-  # For large-scale production use, consider using a caching reverse proxy like
-  # NGINX, varnish or squid.
-  # config.action_dispatch.rack_cache = true
-
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
-  config.serve_static_files = ENV['RAILS_SERVE_STATIC_FILES'].present?
-  config.static_cache_control = 'public, max-age=31536000'
```

Comment:
Do we need this? We could probably get it again by copying the config from `test.rb` below but I think it caused issues for me when I modified TinyMCE's `tinymce.css` – that didn't change in either of my browsers until I manually loaded *it* and cleared the cache on that page, instead of just Ctrl+Shift+R on the main site. (Alternatively, should we look into putting `tinymce.css` into the assets folder, and getting TinyMCE to use the digest properly?)

Reply:
I'm pretty confident we do, yes, because omg will my AWS bill get horrible if this isn't cached. Sounds like we need tinymce.css in the asset folder (and like, probably more things in public/ as well tbh?)

Reply:
I was under the impression this was only for assets in the 'public' folder but I might well be wrong.

I'll add it back in anyway. (But yeah, if we have CSS or JS in public we should probably move them, I'll make an issue.)